src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Data/UpdateAttributes/UpdateAttributes.xaml.cs

```diff
@@ -182,3 +182,5 @@ private async void DamageType_Changed(object sender, SelectionChangedEventArgs e
         }
     }
 }
+
```

Comment:
This is the last instance of this issue I'll point out since it appears to be a script-related issue